spec/unit/caching_spec.rb

```diff
@@ -113,9 +113,9 @@
 
   class TestCache < Hash
     def read(key)
-      if cached = self[key]
-        Marshal.load(cached)
-      end
+      return unless cached = self[key]
```

Comment:
In this case I feel that an extra line in the beginning of the method would make it easier to read.

cached  = self[key]
return unless cached
...